apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -72,7 +72,13 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
         state?.config?.ui?.isPasswordInputValidationEnabled);
 
     const predefinedCategories: any = useMemo(() => {
-        const originalConnectors: any = GovernanceConnectorUtils.getPredefinedConnectorCategories();
+        let originalConnectors: Array<any> = GovernanceConnectorUtils.getPredefinedConnectorCategories();
```

Comment:
AFAIR, we can use `GovernanceConnectorInterface[]` here. Could you please check?

Reply:
`GovernanceConnectorInterface[]` is not compatible the connector category array returned here [1].

[1] https://github.com/wso2/identity-apps/blob/dc45f905593208caebfeccf64055ff2b406f4f54/apps/console/src/features/server-configurations/utils/governance-connector-utils.ts#L177?plain=1